src/site/content/en/blog/app-shortcuts/index.md

```diff
@@ -0,0 +1,233 @@
+**Application** panel of DevTools.
+
+<figure class="w-figure">
+  <img src="./app-shortcuts-devtools.png" alt="Screenshot of app shortcuts in DevTools">
+  <figcaption class="w-figcaption">App shortcuts shown in DevTools</figcaption>
+</figure>
+
+This pane provides a human-readable version of many of your manifest's
+properties, including app shortcuts. It makes it easy to verify that all of the
+shortcut icons, if provided, are loading properly.
+
+{% Aside %}
+DevTools support for app shortcuts is available in Chrome 84 and Edge 84 and later.
+{% endAside %}
+
+App shortcuts may not be available right away to all users because PWA updates
+are capped to once a day.  You can find more details in this [article].
+
+## Best practises
+
+### Order app shortcuts by priority
+
+You are encouraged to order app shortcuts by priority, with the most critical
+app shortcuts appearing first in the `shortcuts` array as the limit on the
+number of app shortcuts displayed varies depending on the platform. Chrome and
+Edge on Windows for instance limit the number of app shortcuts to 10 while
+Chrome for Android only takes the first 4 app shortcuts into account.
+
+### Use distinct app shortcut names
+
+You should not rely on icons to differentiate app shortcuts as they may not
+always be visible. For instance, macOS doesn't support icons in the dock
+shortcuts menu. Use distinct names for each app shortcut.
+
+### Measure app shortcuts usage
+
+You should annotate app shortcuts `url` entries like you would do with
+`start_url` for analytics purposes (.e.g. `url:
+"/my-shortcut?utm_source=homescreen"`).
+
+## Browser support
+
+App shortcuts are available in Chrome 84 on Android. On Windows, you can
+experiment with them on both Chrome 84 and Edge 84 by enabling the flag
+`about://flags/#enable-desktop-pwas-app-icon-shortcuts-menu`. More desktop
+platform support will follow.
+
+## Trusted Web Activity support
+
+[Bubblewrap], the recommended tool to build Android apps that use [Trusted Web
+Activity], reads app shortcuts from the web app manifest and automatically
+generates the corresponding configuration for the Android app.
+
+For developers integrating Trusted Web Activity manually into their Android
+application, [Android app shortcuts] can be used to implement the same
+behaviors.
+
+## Sample
+
+Check out the app shortcuts [sample] and its [source].
+
+<video
+  autoplay
+  loop
+  muted
+  style=" height: 480px; width: 100%"
+  poster="https://storage.cloud.google.com/webfundamentals-assets/videos/app-shortcuts-recording.png"
+  src="https://storage.cloud.google.com/webfundamentals-assets/videos/app-shortcuts-recording.mp4">
+</video>
+
+## Helpful links
+
+* [Explainer][explainer]
+* [Spec][spec]
+* [App shortcuts sample][sample] | [App shortcuts sample source][source]
+* [Tracking bug][cr-bug]
+* [ChromeStatus.com entry][cr-status]
+* Blink Component: [`UI>Browser>WebAppInstalls`]
+
+<!-- lint disable definition-case -->
```

Comment:
@robdodson why do we enforce so many rules on link IDs? Seems kinda obtrusive IMO. I'm pretty sure that 11ty will barf errors if any of the link IDs don't work, so I'm not sure why we're enforcing specific conventions.

Reply:
Our remarklintrc file uses the [remark-preset-lint-markdown-style-guide](https://github.com/remarkjs/remark-lint/tree/master/packages/remark-preset-lint-markdown-style-guide#rules) as a way to set a number of helpful defaults (https://github.com/GoogleChrome/web.dev/blob/master/.remarkrc.js#L8). We used this preset because we didn't know what rules we would want, so it's easier to start with a preset and disable things that you find annoying.

You can turn off this rule globally by adding `['lint-definition-case', false],` to that file.